app/database/queries/breeze_buddy/call_execution_config.py

```
@@ -150,53 +150,47 @@ def insert_call_execution_config_query(
+    """
+    Generate query to get the call execution config owned by a template.
+
+    template_id is the primary linkage between a template and its calling
+    config (1:1); name-based lookup remains only as a legacy fallback for
+    rows created before the link was enforced.
+    """
+    text = f"""
+        SELECT *
+        FROM "{CALL_EXECUTION_CONFIG_TABLE}"
+        WHERE template_id = $1;
+    """
```

Ruff (0.15.20)

[error] Possible SQL injection vector through string-based query construction

(S608)